linux_os/guide/services/snmp/snmp_configure_server/snmpd_not_default_password/tests/missing.pass.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
The OVAL is written as `check_existence="none_exist"` so it is expected to pass if the file doesn't exist because it doesn't find any occurrence. Unless this is not the behavior we want.

Reply:
Aha, I see that in the missing.pass.sh the file is actually not deleted.

Reply:
Updated to remove the configuration file.